bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/FindReplaceLogic.java

```diff
@@ -359,6 +352,7 @@ private boolean performSearch(boolean mustInitIncrementalBaseLocation, String fi
 				// we don't keep state in this dialog
```

Comment:
is this unhandled catch intended?

Reply:
This code is directly copied from https://github.com/eclipse-platform/eclipse.platform.ui/blob/01c5b676196b84ed721cfa43267079374abf4498/bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceDialog.java#L1544

As stated in the code-comment, the dialog does not keep state + the code compiles just as well without this catch-clause.
Even if this was problematic, changing this here is not in the scope of this PR. If you believe this is a big enough poblem, please create an issue for it.